classes/OccurrenceIndividual.php

```diff
@@ -277,11 +277,13 @@ private function setAdditionalIdentifiers(){
 			while($r = $rs->fetch_object()){
 				$identifierTag = $r->identifiername;
 				if(!$identifierTag) $identifierTag = 0;
-				$retArr[$identifierTag][] = $r->identifiervalue;
+				$retArr[$r->idomoccuridentifiers]['name'] = $identifierTag;
+				$retArr[$r->idomoccuridentifiers]['value'] = $r->identifiervalue;
 			}
 			$rs->free();
 		}
-		if($retArr) $this->occArr['othercatalognumbers'] = json_encode($retArr);
+		if($retArr) $this->occArr['othercatalognumbers'] = $retArr;
+		elseif($this->occArr['othercatalognumbers']) $this->occArr['othercatalognumbers'][0]['value'] = $this->occArr['othercatalognumbers'];
```

Comment:
Will this create issues if there's more than one value in `$othercatalognumbers`?